README.md

```diff
@@ -126,6 +126,7 @@ eks_rolling_update.py -c my-eks-cluster
 | INSTANCE_WAIT_FOR_STOPPING | Wait for terminated instances to be in `stopping` or `shutting-down` state as well as `terminated` or `stopped`       | False                                    |
 | BATCH_SIZE                 | Instances to scale the ASG by at a time. When set to 0, batching is disabled.                                         | 0                                        |
 | ASG_NAMES                 | List of space-delimited ASG names. Out of ASGs attached to the cluster, only these will be processed for rolling update. If this is left empty all ASGs of the cluster will be processed. | "" |
+| RESTART_DEPLOYMENT        |  If true, discover single replica pods on the node and perform a rollout restart | False |
```

Comment:
In that file, do you mind mentioning on line 34 the optional deployment restart for single pods

Reply:
Thanks @aquam8 I made the comment